whats_fresh/whats_fresh_api/views/entry/images.py

```diff
@@ -31,7 +33,21 @@ def image_list(request):
     elif request.GET.get('saved') == 'true':
         message = "Entry saved successfully!"
 
-    paginator = Paginator(Image.objects.order_by('name'), settings.PAGE_LENGTH)
+    if request.GET.get('search') is None or request.GET.get('search') == "":
+        images = Image.objects.order_by('name')
+    else:
+        if request.GET.get('search') != "":
+            images = list(OrderedDict
```

Comment:
these indentations are getting a little silly :P Is there a way to do this that doesn't have the content_auto indented 50 characters?